package.json

```diff
@@ -116,9 +117,10 @@
     "ovh-angular-responsive-tabs": "^4.0.0",
     "ovh-angular-sidebar-menu": "^7.2.2",
     "ovh-angular-swimming-poll": "^3.0.0",
+    "ovh-angular-tail-logs": "^1.1.1",
```

Comment:
I roughly encourage you to upgrade to the latest version.

```
sh
yarn remove ovh-angular-tail-logs
yarn add @ovh-ux/ng-tail-logs
```

In that way you can declare only one import statement from the `client/app/app.js` (and get rid of the import from the `client/app/index.js`.

Furthermore you can refer to the Telecom Control Panel implementation which is already uses tail-log components in its latest version (https://github.com/ovh-ux/ovh-manager-telecom/blob/master/client/app/app.js#L5).

ref: https://github.com/ovh-ux/ng-tail-logs/blob/master/CHANGELOG.md#breaking-changes

Reply:
I suceed to update the code to ng-tail-logs v2.0.0-beta.1, however the logs are duplicated at each poll.

1st poll: 1 log
2nd poll: 2 logs (2 time the same)
3rd poll: 3 logs (3 times the same)

I suspect a bug in the new ng-tail-logs release.
If it can help i can also commit my changes on this PR but it will render the code not functionnal.

Reply:
Thanks for the update @pdepaepe.

Since it's already validated from the QA Team then I will handle the bump in a separate pull request.